README.md

```diff
@@ -22,5 +22,5 @@ From within the addon directory, run `gulp`.
 # License
 MIT
 
-[soundcloud-link]: https://soundcloud.com/angelarenaallstars "Music for Angel Arena Allstars"
-[discord-link]: https://discord.gg/ "Angel Arena Allstars Discord Instant Invite"
+[soundcloud-link]: https://soundcloud.com/angelarenaallstars "Music for Open Angel Arena"
+[discord-link]: https://discord.gg/ "Open Angel Arena Discord Instant Invite"
```

Comment:
we should also just add the real discord link right into the description. it's already old news so i'm not worried about spam